templating.rst

```diff
@@ -51,7 +51,7 @@ template - a text file parsed by PHP that contains a mix of text and PHP code:
 
 .. index:: Twig; Introduction
 
-But Symfony packages an even more powerful templating language called `Twig`_.
+But Symfony packages have even more powerful templating language called `Twig`_.
```

Comment:
I think this change is not OK. "packages" here doesn't mean *"software libraries"* but the verb *"includes"*

Reply:
Thanks, I fixed that.